src/modules/recipes/models.ts

```diff
@@ -0,0 +1,62 @@
+export interface Recipe {
+  idMeal: string;
+  strMeal: string;
+  strMealThumb: string;
+  strCategory?: string;
+  strArea?: string;
+  ingredients: string[];
+  strInstructions: string;
+  strIngredient1?: string;
+  strIngredient2?: string;
+  strIngredient3?: string;
+  [key: `strIngredient${number}`]: string | undefined;
```

Comment:
Check this API response as an example. The goal is to make sure we have all the properties from the response in this model: https://www.themealdb.com/api/json/v1/1/lookup.php?i=52772

Regarding the dynamic key (`[key: `strIngredient${number}`]: string | undefined;`), I'd suggest to avoid it in this situation as we want to make sure the we only have 20 ingredients and 20 measures.